src/firewall/core/nftables.py

```diff
@@ -1217,7 +1217,7 @@ def _rule_addr_fragment(self, addr_field, address, invert=False):
         else:
             if check_mac(address):
                 if addr_field == "daddr":
-                    raise FirewallError(INVALID_RULE, "%s._rule_addr_fragment()", (self.__class__))
+                    return ""
```

Comment:
I think this should return `{}` instead of `""`. That's what other functions don that are support to return a fragment.